com.reprezen.swagedit/src/com/reprezen/swagedit/editor/hyperlinks/JsonReferenceHyperlinkDetector.java

```diff
@@ -58,13 +58,19 @@ protected boolean canDetect(String basePath) {
 			}
 			return new IHyperlink[] {
 					new SwaggerHyperlink(reference.getPointer().toString(), viewer, info.region, target) };
-		} else {
-			IFile file = DocumentUtils.getWorkspaceFile(baseURI.resolve(reference.getUri()));
-			if (file != null && file.exists()) {
-				return new IHyperlink[] {
-						new SwaggerFileHyperlink(info.region, info.text, file, pointer(reference.getPointer())) };
-			}
-		}
+        } else {
+            URI resolved;
```

Comment:
Why are we seeing a difference in the indent level?

This was a red herring w.r.t. to the referenced bug, ZEN-2716.  But it's clearly improved, more robust code that will avoid the unrelated issue observed in the error log.